### PR TITLE
parsing data on project areas table

### PR DESCRIPTION
**Pull request by pflopez**
Still waiting on confirmation for the exact format on the table, matching the format of this screenshot. We have tests for this if we get confirmation we can update accordingly. 

Screen Shot 2023-09-20 at 13 33 20

### CODE REVIEW

File: src/interface/src/app/plan/create-scenarios/create-scenarios.component.html [hunk 1]
```diff
@@ -56,11 +56,12 @@
         <mat-tab label="Results">
           <ng-container [ngSwitch]="scenarioState">
             <app-scenario-not-started
-              *ngSwitchCase="'not-started'"></app-scenario-not-started>
+              *ngSwitchCase="'PENDING'"></app-scenario-not-started>
             <app-scenario-pending
```
[the remark this answers is not in the document]
Comment (sachiyuan) [reply src/interface/src/app/plan/create-scenarios/create-scenarios.component.html:60]: Oh actually, do we want to show app-scenario-not-started ("You need to generate the scenario first to get results") if the status is PENDING? Once it's PENDING that means the scenario has already been generated

[the remark this answers is not in the document]
Comment (pflopez) [reply src/interface/src/app/plan/create-scenarios/create-scenarios.component.html:60]: not sure i follow, the status are :
 'PENDING'  | 'RUNNING'  | 'SUCCESS'  | 'FAILURE'

Shouldn't `RUNNING` be the one meaning that is being generated, and `SUCCESS` once it's done?

[the remark this answers is not in the document]
Comment (sachiyuan) [reply src/interface/src/app/plan/create-scenarios/create-scenarios.component.html:58]: For documentation's sake, the outcome of our conversation:
Screenshot 2023-09-22 at 1 02 58 PM

File: src/interface/src/app/plan/create-scenarios/create-scenarios.component.ts [hunk 2]
```diff
@@ -85,8 +87,10 @@ export class CreateScenariosComponent implements OnInit, OnDestroy {
   private readonly destroy$ = new Subject<void>();
   project_area_upload_enabled = features.upload_project_area;
 
-  // TODO This should come from somewhere
-  scenarioState: ScenarioState = 'not-started';
+  // this value gets updated once we load the scenario result.
+  scenarioState: ScenarioResultStatus = 'PENDING';
```
[the remark this answers is not in the document]
Comment (sachiyuan) [reply src/interface/src/app/plan/create-scenarios/create-scenarios.component.ts]: Resolved in another conversation (decision captured in a previous thread)

**Review by sachiyuan — APPROVED** [review]
LGTM!

**Pull request merged**